helpers/lbryHelpers.js

```diff
@@ -1,6 +1,7 @@
 var path = require('path');
```

Comment:
may have gone a little overboard, but I did a big reorganize to handle this and hopefully lay the groundwork for better organization going forward.  Separated out true controller functions from helper methods.